python_modules/dagster-graphql/dagster_graphql/schema/external.py

```diff
@@ -107,6 +107,37 @@ class Meta:
         name = "RepositoryLocationOrLoadError"
 
 
+class GrapheneWorkspaceLocationStatusEntry(graphene.ObjectType):
+    id = graphene.NonNull(graphene.ID)
+    name = graphene.NonNull(graphene.String)
+    loadStatus = graphene.NonNull(GrapheneRepositoryLocationLoadStatus)
+
+    class Meta:
+        name = "WorkspaceLocationStatusEntry"
+
+    def __init__(self, name, load_status):
+        super().__init__(name=name, loadStatus=load_status)
+
+    def resolve_id(self, _):
+        return self.name
```

Comment:
is this what the other workspace type uses?

Reply:
yeah